tree/dataframe/src/RDFNodes.cxx

```diff
@@ -219,20 +214,20 @@ unsigned int TSlotStack::GetSlot()
 }
 
 RLoopManager::RLoopManager(TTree *tree, const ColumnNames_t &defaultBranches)
-   : fTree(std::shared_ptr<TTree>(tree, [](TTree *) {})), fDefaultColumns(defaultBranches),
+   : RNode(this), fTree(std::shared_ptr<TTree>(tree, [](TTree *) {})), fDefaultColumns(defaultBranches),
```

Comment:
This looks weird... Would it make sense to have that `fLoopManager` behave as `fLoopManagerUnlessThis = nullptr`, and then simply use the default ctor? E.g. what happens if you copy / move `this`? Then `fLoopManager` would point to the wrong / old "thing", will "nullptr thus this" will work just fine.

Reply:
I agree, this weird.
It might not be a problem because nodes are effectively immutable: once they are added to the computation graph they stay there, set in stone, until they are pruned from the graph or the whole graph is destroyed.
In any case I will change it as you suggest since it's more robust to future changes :)

Reply:
Fixed.